### PR TITLE
Abdul/foreground app with background app

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for ForegroundAppwithBackgroundApp & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : MyBackgroundApp.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : MyForegroundApp.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

1) file modified : MyBackgroundApplicationCpp.vcxproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.
    PlatformToolSet from v141 to v142

2) file modified : MyForegroundAppCpp.vcxproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.
    PlatformToolSet from v141 to v142

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Added the BIOS setting for UPBOARD Pin configuration
    Added the app *packages installing steps

### CODE REVIEW

File: ForegroundAppWithBackgroundApp/CS/MyForegroundApp/Package.appxmanifest [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:iot="http://schemas.microsoft.com/appx/manifest/iot/windows10"
+  IgnorableNamespaces="uap mp iot">
+
+  <Identity
+    Name="MyForegroundApplication-uwp"
+    Publisher="CN=Foreground"
```
Comment (neilsh-msft) [inline ForegroundAppWithBackgroundApp/CS/MyForegroundApp/Package.appxmanifest]: Please change to ' Publisher="CN=MSFT" ' to keep it consistent with other appxmanifest files.

File: ForegroundAppWithBackgroundApp/CPP/MyForegroundAppCpp/Package.appxmanifest [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:iot="http://schemas.microsoft.com/appx/manifest/iot/windows10"
+  IgnorableNamespaces="uap mp iot">
+
+  <Identity
+    Name="MyForegroundAppCpp"
+    Publisher="CN=paulmon"
```
Comment (neilsh-msft) [inline ForegroundAppWithBackgroundApp/CPP/MyForegroundAppCpp/Package.appxmanifest]: Please change to ' Publisher="CN=MSFT" ' to keep it consistent with other appxmanifest files.

File: ForegroundAppWithBackgroundApp/CPP/MyBackgroundApplicationCpp/Package.appxmanifest [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+<Package
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:iot="http://schemas.microsoft.com/appx/manifest/iot/windows10"
+  IgnorableNamespaces="uap mp iot">
+
+  <Identity
+    Name="MyBackgroundApplicationCpp-uwp"
+    Publisher="CN=paulmon"
```
Comment (neilsh-msft) [inline ForegroundAppWithBackgroundApp/CPP/MyBackgroundApplicationCpp/Package.appxmanifest]: Please change to ' Publisher="CN=MSFT" ' to keep it consistent with other appxmanifest files.  Will this also require changing the .pfx file?

File: ForegroundAppWithBackgroundApp/CPP/MyBackgroundApplicationCpp/MyBackgroundApplicationCpp.vcxproj [hunk 4]
```diff
@@ -0,0 +1,234 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{c76e0ce0-3a70-4b30-a6ee-b22af4999f46}</ProjectGuid>
+    <RootNamespace>MyBackgroundApplicationCpp</RootNamespace>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
```
Comment (neilsh-msft) [inline ForegroundAppWithBackgroundApp/CPP/MyBackgroundApplicationCpp/MyBackgroundApplicationCpp.vcxproj]: Should this be "10.0.19041.0"?

File: ForegroundAppWithBackgroundApp/CPP/MyForegroundAppCpp/MyForegroundAppCpp.vcxproj [hunk 5]
```diff
@@ -0,0 +1,195 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>{171eb3ea-c0fd-423d-a3b3-44ff528a7591}</ProjectGuid>
+    <RootNamespace>MyForegroundAppCpp</RootNamespace>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+    <AppContainerApplication>true</AppContainerApplication>
+    <ApplicationType>Windows Store</ApplicationType>
+    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0</WindowsTargetPlatformVersion>
```
Comment (neilsh-msft) [inline ForegroundAppWithBackgroundApp/CPP/MyForegroundAppCpp/MyForegroundAppCpp.vcxproj]: Should this be 10.0.19041.0 instead of 10.0 ?

**Review by neilsh-msft — CHANGES_REQUESTED** [review]

**Review by rsameser — COMMENTED** [review]
Please remove the main ReadMe file - do not update

File: README.md [hunk 6]
```diff
@@ -1,74 +1,109 @@
...
-
-## Samples by category
-
-### Azure IoT Edge
-
-| Name           | Description      |  
-|----------------|------------------|  
-| [interop-textmsg-consoleapp](https://aka.ms/WinIoTSamples-interop-textmsg-consoleapp) | Basic interop sample demonstrating text messaging between a Windows console app and an Edge module. |
-| [interop-customvision-textmsg-uwpapp](https://aka.ms/WinIoTSamples-interop-customvision-textmsg-uwpapp) | <p>Two more advanced interop samples which demonstrate bidirectional communication between a Windows application and an Edge module. </p><ul><li>Text messaging between a UWP application and an Edge module. </li><li>A 'Custom vision' machine learning interop sample with a fruit classifier which uses a Windows UWP app to send camera frames to an Edge module for identification.</li></ul>|  
+# "Hello, blinky!" background service
```
Comment (rsameser) [inline README.md]: Please do NOT update the main ReadMe with the sample app read me

**Review by rsameser — CHANGES_REQUESTED** [review]
Please address comments above

File: ForegroundAppWithBackgroundApp/CPP/MyBackgroundApplicationCpp/MyBackgroundApplicationCpp.vcxproj (hunk 4 above)
Comment (v-abdrahman) [reply ForegroundAppWithBackgroundApp/CPP/MyBackgroundApplicationCpp/MyBackgroundApplicationCpp.vcxproj]: Updated the PlatformVersion from 10.0 to 10.0.19041.0

File: ForegroundAppWithBackgroundApp/CPP/MyForegroundAppCpp/MyForegroundAppCpp.vcxproj (hunk 5 above)
Comment (v-abdrahman) [reply ForegroundAppWithBackgroundApp/CPP/MyForegroundAppCpp/MyForegroundAppCpp.vcxproj]: Updated the PlatformVersion from 10.0 to 10.0.19041.0

File: ForegroundAppWithBackgroundApp/CPP/MyForegroundAppCpp/Package.appxmanifest (hunk 2 above)
Comment (v-abdrahman) [reply ForegroundAppWithBackgroundApp/CPP/MyForegroundAppCpp/Package.appxmanifest]: Changed the Publisher name as requested "CN=MSFT"

File: ForegroundAppWithBackgroundApp/CS/MyForegroundApp/Package.appxmanifest (hunk 1 above)
Comment (v-abdrahman) [reply ForegroundAppWithBackgroundApp/CS/MyForegroundApp/Package.appxmanifest]: Changed the Publisher name as requested "CN=MSFT"

File: ForegroundAppWithBackgroundApp/CPP/MyBackgroundApplicationCpp/Package.appxmanifest (hunk 3 above)
Comment (v-abdrahman) [reply ForegroundAppWithBackgroundApp/CPP/MyBackgroundApplicationCpp/Package.appxmanifest]: Changed the Publisher name as requested "CN=MSFT"

File: README.md (hunk 6 above)
Comment (rsameser) [reply README.md]: PLEASE REMOVE THIS FILE - it should not have been updated

Comment (v-abdrahman) [reply README.md]: Restored the same readme file which was present in the window iot repository at "Foreground app with Background app" sample's main readme page.

Comment (rsameser) [reply README.md]: This is still here...

**Review by neilsh-msft — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (v-abdrahman) [reply README.md]: Its done.

**Pull request merged**